my-app/src/components/Navigation.js

```diff
@@ -0,0 +1,176 @@
+import React from "react";
+import { useNavigate, NavLink, useLocation } from "react-router-dom";
+import Logout from "../../src/images/logout.svg";
+import blackLogout from "../../src/images/logout-black.svg";
+
+function Navigation(props) {
+  const location = useLocation();
+  const { username, loggedIn, setLoggedIn, isMenuOpen, isMobile } = props;
+  const navigate = useNavigate();
+
+  function handleSignOut() {
+    localStorage.removeItem("token");
+    window.location.reload();
+    setLoggedIn(false);
+    navigate("/");
+  }
+
+  return (
+      <nav className="navigation">
+        {props.isMobile ? (
+          <div className="navigation__main">
+            <div className="navigation__subwrapper">
+            {!props.isBackPopupOpen && (
+  <button
+    className="navigation__button"
+    type="button"
+    onClick={props.prepareHamburger}
+  >
```

Comment:
* You need to format the markup to show indentation and to make the code more readable. You can do that with `prettier` or a combination `SHIFT + ALT + F`